airflow-postgres-app-services/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "airflowWebAppName": {
+      "value": "airflow-postgres-appservices"
+    },
+    "postgreDatabaseLogin": {
+      "value": "pgAdmin"
+    },
+    "postgrePassword": {
+      "value": "pgP@ssw0rd!"
+    },
+    "postgreVersion": {
```

Comment:
remove this and use a defaultValue in the template

Reply:
Updated!